go.mod

```diff
@@ -1,14 +1,27 @@
 module github.com/aws/ec2-macos-utils
 
-go 1.16
+go 1.21
```

Comment:
Had a quick chat with Matt - we don't expect to need 1.21 and will see if we can lower the requirement until 1.21 is more widely available/adopted.

Will post back here with an update, a bit later today.

Reply:
Done